undertow/src/main/java/org/wildfly/extension/undertow/ServerAdd.java

```diff
@@ -38,16 +38,16 @@
 /**
  * @author <a href="mailto:tomaz.cerar@redhat.com">Tomaz Cerar</a> (c) 2013 Red Hat Inc.
  */
-class ServerAdd extends AbstractBoottimeAddStepHandler {
+class ServerAdd extends AbstractAddStepHandler {
```

Comment:
@stuartwdouglas is this ok? It's always nice to allow adds post-boot, but I wanted to confirm with you that you want to support that.